docs/ci.md

```diff
@@ -0,0 +1,44 @@
+# CI
+
+## Parallelization with Parallel & Knapsack
+Knapsack and Parallel gems allow us to run tests in several nodes at the same time, benefiting us in the execution time. Knapsack parallelizes them at node level while Parallel does it at CPU level.
+
+Knapsack splits tests based on an execution time report. In case there are files that were not added in the report, they will all run on the same node and may overload it, so it is strongly recommended to update the report frequently.
+
+## Configuration
+In case you want to use this you will need a script that split spec files called `parallel_tests` which sets up the configuration, assuming you have `n_nodes * cpu_cores_quantity`.
+
+On Github Actions you can add any nodes you want using matrix strategy, setting up some variables:
+
+```sh
+  ci_node_total: [4]
+  # set N-1 indexes for parallel jobs
+  # When you run 2 parallel jobs then first job will have index 0, the second job will have index 1 etc
+  ci_node_index: [0, 1, 2, 3]
+```
+
+CPU cores quantity on every node are obtained automatically from Github Actions config `echo "cpu_cores=$(nproc)" >> $GITHUB_ENV`.
+
+If you want to update it manually you can do it by updating this variable:
+`PARALLEL_TESTS_CONCURRENCY: 2`
```

Comment:
I don't think someone would want to do this 🤔

Reply:
I'm not sure whether to remove it; it might be useful if the machine has many cores to limit parallelism or to deactivate it temporarily, wdyt?